int64 count
DetectionBox[] detections
